### DISCUSSION TITLE
Recommend a featured article

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#126] [Content]
On the homepage of MDN Web Docs, we show 4 feature articles. Ideally, we want to rotate these on a monthly basis but, should always have at least one HTML, CSS, and JavaScript article. If there are any you would like to recommend, please use this discussion thread to do so.

**Comment by MrBrain295** [comment]
[This page](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) has helped me quite a bit.
